proto/api/api.proto

```diff
@@ -16,7 +16,7 @@ import "proto/types/action.proto";
 import "proto/types/blockchain.proto";
 import "proto/types/node.proto";
 import "proto/types/election.proto";
-import "proto/types/transaction_log.proto";
+import "proto/types/system_log.proto";
```

Comment:
let's not change the file name? to be consistent with iotex-core, where it is also named TransactionLog